core/tests/webdriverio_utils/ExplorationEditorMainTab.js

```diff
@@ -817,13 +818,11 @@ var ExplorationEditorMainTab = function() {
     await action.waitForAutosave();
     await general.scrollToTop();
     var nodeStateElement = await explorationGraph.$(
-      `.e2e-test-node=${stateName}`);
+      `.e2e-test-node*=${stateName}`);
```

Comment:
Why *?

Reply:
Here * is needed because the stateName is not the complete text present inside the e2e-test-node element. And for patial text we use `*=`